src/library/FauxControl/styled.js

```diff
@@ -137,7 +137,11 @@ export const Underlay = styled('div', {
     position: 'absolute',
     right: 0,
     top: 0,
-    zIndex: -1
+    zIndex: -1,
+
+    /* TargetX Custom Styles */
+    borderTopWidth: 0,
```

Comment:
Let's remove these for now. I'm not comfortable or convinced that Ryan wants that box shadow.